.gitignore

```diff
@@ -121,10 +119,10 @@ scala-package/*/*/target/
 .pydevproject
 CMakeFiles
 cmake_install.cmake
-dmlc-core
-ps-lite
-nnvm
-!src/nnvm
+#dmlc-core
```

Comment:
Why would these be ignored? They are submodules. CLion gives warnings about this and treats the subdirectories strangely.